_episodes/14.md

```diff
@@ -0,0 +1,330 @@
+---
+layout: episode
+title:  "14: Iceberg: March of the Trinos."
+date: 2021-04-01
+tags: iceberg apache datalake table format sql schema evolution partition
+youtube_id: "CEKz8JvfxuE"
+wistia_id: "2ef3k68qez"
+sections: 
+   - title: "Concept of the week"
+     desc: "Apache Iceberg and the table format"
+     time: 1034
+   - title: "PR of the week"
+     desc: "PR 1067 Add Iceberg connector"
+     time: 3562
+   - title: "Demo"
+     desc: "Creating tables through Iceberg and reading them through Trino"
+     time: 3844
+   - title: "Question of the week"
+     desc: "Why do we still depend on the Hive metastore if metadata for
+      Iceberg saves to the filesystem?"
+     time: 4719
+---
+
+Trino nation, We want to hear from you! If you have a question or pull request 
+that you would like us to feature on the show please join the 
+[Trino slack](slack.html), and go to the \#trino-community-broadcast channel 
+and let us know there. Otherwise, you can message Manfred Moser or Brian Olsen 
+directly. Also, feel free to reach out to us on our Twitter channels Brian 
+[@bitsondatadev](https://twitter.com/bitsondatadev) and Manfred 
+[@simpligility](https://twitter.com/simpligility).
+
+If you want to show us some 💕, please [give us a ⭐ on Github](/star).
+
+<p align="center">
+<img align="center" width="50%" height="100%" src="/assets/episode/14/trino-penguin.png"/><br/>
+March of the Trinos! Be careful Commander Bun Bun! That Iceberg doesn't look stable!<br/>
+<a href="https://joshdata.me/iceberger.html">https://joshdata.me/iceberger.html</a>
+</p>
+
+## Iceberg links
+
+ * [Apache Iceberg](https://iceberg.apache.org/)
+ * [Apache Iceberg Community](https://iceberg.apache.org/community/)
+
+## Guests
+  
+ * David Phillips, creator of Trino and Presto, and CTO at 
```

Comment:
"creator of Trino/Presto" .. otherwise this looks like they are different tools..